### ISSUE TITLE
Typescript: Do not include type definitions (Rollup import conflict)

### ISSUE THREAD

**Issue by scarejar** [axios/axios#1571]
#### Summary

In my app I already have 
<script src = "axios.js"></script>
<script src = "bundle.js"></script>

I'm using rollup to bundle my app so doing 
`import axios from 'axios';`
does not work because rollup tries to bundle typescript.

I'm working around it via:
In ts.config, there's already a types property that allows anyone to import types with ease so I removed all exports from the type definition file and `declare var Axios: AxiosStatic;` at the bottom.

tl;dr, remove the feature to import type definitions

####
 Context

- axios version:  v17"
- Environment: node v10, chrome 66, Arch Linux*

**Comment by chinesedfan** [comment]
I think you can find a better rollup config, instead of requesting axios to remove the type definitions. Good luck.
